system-addon/content-src/components/Card/Card.jsx

```diff
@@ -194,6 +270,14 @@ export class _Card extends React.PureComponent {
    </li>);
   }
 }
-_Card.defaultProps = {link: {}};
+
+_Card.defaultProps = {
+  link: {},
+  url: global.URL || {
```

Comment:
I'm guessing this is only for testing? Any reason not to just override the global for the test instead of passing in special props?

Reply:
Yes, that's right. It's just for testing. I modified the code to override the global for the test because it looks cleaner after thinking about it.